meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,312 @@
+  ]'
+```
+
+### Query Meilisearch using Vectors
+
+Use the new `vector` search parameter with the `/search` and `/multi-search` to search for documents with the nearest vector. You must compute the vector query with a third-party tool.
+
+```bash
+curl -X POST -H 'content-type: application/json' \
+'localhost:7700/indexes/songs/search' \
+--data-binary '{ "vector": [0, 1, 2] }'
+```
+
+### Similarity score
+
+When you use vector search, returned documents include a `_semanticScore` field.
+
+```json
+{
+  "hits": [
+    { "id": 0, "_vectors": [0, 0.8, -0.2], "title": "Across The Universe", "_semanticScore": 0.6754 },
+    { "id": 1, "_vectors": [1, -0.2, 0], "title": "All Things Must Pass", "_semanticScore": 0.7546 },
+    { "id": 2, "_vectors": [[0.5, 3, 1], [-0.2, 4, 6]], "title": "And Your Bird Can Sing", "_semanticScore": 0.78 }
+  ],
+  "query": "",
+  "vector": [0, 1, 2],
+  "processingTimeMs": 0,
+  "limit": 20,
+  "offset": 0,
+  "estimatedTotalHits": 2
+}
+```
+
+
+🗣️ This feature is experimental and we need your help to improve it! Share your thoughts and feedback on [this GitHub discussion](https://github.com/meilisearch/product/discussions/677).
+
+⚠️ Experimental features may be incompatible between Meilisearch versions.
+
+Done by @Kerollmops in #3825
+
+## Display ranking scores at search
+
+Use the new `showRankingScore` search parameter to see the ranking scores for returned documents. 
+
+```bash 
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "Batman Returns", "showRankingScore": true }'
+```
+
+Each returned document will include a `_rankingScore` property displaying a score between 0 and 1. The higher the ranking score, the more relevant the document.
+
+```json 
+"_rankingScore": 0.8575757575757575,
+```
+
+### Ranking score details — Experimental 
+
+View detailed scores per ranking rule for each document with the experimental `showRankingScoreDetails` search parameter. 
+
+```bash 
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "Batman Returns", "showRankingScoreDetails": true }'
+```
+
+When `showRankingScoreDetails` is set to `true`, returned documents include a `_rankingScoreDetails` field. This field contains score values for each ranking rule.
+
+```json 
+"_rankingScoreDetails": {
+  "words": {
+    "order": 0,
+    "matchingWords": 1,
+    "maxMatchingWords": 1,
+    "score": 1.0
+  },
+  "typo": {
+    "order": 1,
+    "typoCount": 0,
+    "maxTypoCount": 1,
+    "score": 1.0
+  },
+  "proximity": {
+    "order": 2,
+    "score": 1.0
+  },
+  "attribute": {
+    "order": 3,
+    "attributes_ranking_order": 0.8,
+    "attributes_query_word_order": 0.6363636363636364,
+    "score": 0.7272727272727273
+  },
+  "exactness": {
+    "order": 4,
+    "matchType": "noExactMatch",
+    "matchingWords": 0,
```

Comment:
@dureuill just highlighting I've added this based on the latest PR. any issue let me know